### PR TITLE
Internal load balancer changes and binary added to work with marketplace

### PR DESCRIPTION
**Pull request by rjaini**
Reason for Change:
fix:Added load balancer code and binary in example folder to work for Azure stack market place.
Issue Fixed:
#1
Notes:
Update the code with workaround to support internal load balancer.

### CODE REVIEW

**Comment by rjaini** [comment]
@honcao @vikasnav @jadarsie @deaborch

File: parts/k8s/kubernetesagentcustomdata.yml [hunk 1]
```diff
@@ -211,7 +211,7 @@ write_files:
     - name: localcluster
       cluster:
         certificate-authority: /etc/kubernetes/certs/ca.crt
-        server: https://{{WrapAsVariable "kubernetesAPIServerIP"}}:443
+        server: https://<kubernetesAPIServerIP>:443
```
Comment (jadarsie) [inline parts/k8s/kubernetesagentcustomdata.yml]: You sure you do not need WrapAsVariable here?

Comment (jadarsie) [reply parts/k8s/kubernetesagentcustomdata.yml]: Disregard, I saw it is needed later

File: parts/k8s/kubernetesmasterresources.t [hunk 2]
```diff
@@ -194,8 +180,22 @@
                 "sourceAddressPrefix": "10.0.0.0/8",
                 "sourcePortRange": "*"
               }
-            }
+            },
```
Comment (honcao) [inline parts/k8s/kubernetesmasterresources.t]: remove , here and add , inside if IsAzureStackCloud

File: parts/k8s/kubernetesmasterresources.t [hunk 3]
```diff
@@ -194,8 +180,22 @@
...
         {{end}}
+        {
+            "name": "allow_kube_tls",
```
Comment (honcao) [inline parts/k8s/kubernetesmasterresources.t]: {{if IsAzureStackCloud}}
            ,{
              "name": "allow_vnet_inbound",
              "properties": {
                "access": "Allow",
                "description": "Allow traffic to specific addresses.",
                "destinationAddressPrefix": "10.0.0.0/8",
                "destinationPortRange": "*",
                "direction": "Inbound",
                "priority": 4095,
                "protocol": "*",
                "sourceAddressPrefix": "10.0.0.0/8",
                "sourcePortRange": "*"
              }
            },
            {
              "name": "allow_vnet_outbound",
              "properties": {
                "access": "Allow",
                "description": "Allow traffic to specific addresses.",
                "destinationAddressPrefix": "10.0.0.0/8",
                "destinationPortRange": "*",
                "direction": "Outbound",
                "priority": 4095,
                "protocol": "*",
                "sourceAddressPrefix": "10.0.0.0/8",
                "sourcePortRange": "*"
              }
            }
        {{end}}

File: parts/k8s/kubernetesagentcustomdata.yml (hunk 1 above)
Comment (honcao) [reply parts/k8s/kubernetesagentcustomdata.yml]: Yes. This is placeholder and it will be replaced in the script.

**Pull request merged**